tt_submission/meta.yaml

```
@@ -0,0 +1,15 @@
+title: "Historical and modern seasonal forecasting: Comparison of the 'Summer Forecast' of Zurich's Böögg with a Dynamic Climate Model "
```

```suggestion
title: "Seasonal Forecasting with Zurich's Böögg"
```
`:` can be dangerous inside yaml (since it has meaning), and I generally like to keep the titles at least relatively short. This long title might break social media posts. Think of the title as a phrase to go into "This week we're exploring {title}" (because that's part of how I use it).